terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/date/DateConvertUtils.java

```diff
@@ -93,4 +92,15 @@ public static Time convertToTime(java.util.Date date) {
         Time time = new Time(cal.getTimeInMillis());
         return time;
     }
+
+    /**
+     * Check java.util.Date is not null<br>
+     * @param date check target
+     * @throws IllegalArgumentException if the date is null
+     */
+    private static void CheckDate(java.util.Date date) {
+        if (date == null) {
+            throw new IllegalArgumentException(MESSAGE_ARGUMENT_NULL_ERROR);
```

Comment:
- [ ] No need to define constant value.

Reply:
fix on 1103be3f53d29d2d92ebf7396a1599d791e67bf5